src/CONST.ts

```diff
@@ -1463,6 +1463,16 @@ const CONST = {
             DISABLE: 'disable',
             ENABLE: 'enable',
         },
+        COLLECTION_FIELDS: {
```

Comment:
not sure about this const name, but also cannot think what would be the best here. maybe COLLECTION_KEYS, FEATURE_KEYS